docs/openapi/index.yaml

```diff
@@ -122,7 +122,26 @@ components:
         activityRecipientId:
           type: number
         name:
+<<<<<<< HEAD
```

Comment:
Looks like an unresolved merge conflict here

Reply:
Whoops, good catch. Should be fixed now.